pkgs/tools/misc/linux-logo/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gettext
+, nix-update-script
+}:
+
+stdenv.mkDerivation rec {
+  pname = "linux_logo";
+  version = "6.0";
+
+  src = fetchFromGitHub {
+    owner = "deater";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-q8QznEgnALJS//l7XXHZlq07pI2jCCm2USEU96rO8N0=";
+  };
+
+  nativeBuildInputs = [ gettext ];
+
+  postPatch = ''
+    sed -i configure \
```

Comment:
`substituteInPlace` is preferable because it warns on failure.

Reply:
Yes, but it doesn't allow using regexp like I can via `sed`. Anyway, I've found a way to remove `postPatch`.